src/vector/index.html

```diff
@@ -22,6 +22,21 @@
     <meta name="msapplication-config" content="<%= require('../../res/vector-icons/browserconfig.xml') %>">
     <meta name="theme-color" content="#ffffff">
     <meta property="og:image" content="<%= htmlWebpackPlugin.options.vars.og_image_url %>" />
+    <meta http-equiv="Content-Security-Policy" content="
+        default-src 'none';
+        style-src 'self' 'unsafe-inline';
+        script-src 'self' 'unsafe-eval' https://www.recaptcha.net https://www.gstatic.com;
+        img-src * blob: data:;
```

Comment:
Why `img-src *`?

At the time [I've suggested one](https://github.com/vector-im/riot-web/issues/3632#issuecomment-367110054) it did seem to work without that (in my tests). Maybe I did not use/test some third-party image embedding loading?

That said, loading images directly from third-party servers may/should possibly be avoided due to privacy concerns, should not it?

Reply:
it was copied from the CSP header served at riot.im/app
I did not evaluate img-src, feel free to open an issue for it

Reply:
done: https://github.com/vector-im/riot-web/issues/12304